### PR TITLE
Tab first branch

### PR DESCRIPTION
**Pull request by Tabermory**
Hope I`m doing right)

### CODE REVIEW

**Comment by artem-trubin** [comment]
Hey, there are a lot of different changes here. Check `Files changed` tab to make sure only your changes are there. Remove those `<<<<<<< HEAD` and stuff like this.

File: README.md [hunk 1]
```diff
@@ -112,13 +112,15 @@ Good luck everyone :raised_hands:
 
 # Hello Kottans!
 
-
 # Hey there!
 
+<<<<<<< HEAD
```
Comment (suchov) [inline README.md:117]: this is part of the conflict resolving, should be deleted

File: README.md [hunk 2]
```diff
@@ -127,24 +129,29 @@ I hope this project will be of interest to each of us.
...
 
 ## Hi guys!
+
 My name is ~~Pablo Escobar~~ Pavlo))
 My [repo](https://github.com/zenoksi4/kottans-frontend).
 My cat it so cute)
 ![cat_pablo_escobar](\assets\images\my_cate_so_cute.png)
 
 # Hello, Kottans!
+<<<<<<< HEAD
```
Comment (suchov) [inline README.md:149]: This one too...

File: README.md [hunk 3]
```diff
@@ -258,15 +264,13 @@ My name is Oleksandr and I am happy to study with all of you!
...
-
 #Hi! My name is Di and its my first steps in IT.
 
-
 # Hello, kottans!
+
 My name is Oleg. I'm switcher )). I'm glad to join the kottans community.
 Link to my repo [here](https://https://github.com/OlegPopovych/kottans-frontend).
 
-
```
Comment (suchov) [inline README.md:269]: no need for empty line deletion

**Review by suchov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**